include/cetl/cetl.h

```diff
@@ -0,0 +1,41 @@
+/// @file
+/// CETL common header.
+///
+/// Keep this very spare. CETL's desire is to adapt to future C++ standards
+/// and too many CETL-specific definitinos makes it difficult for users to switch off of CETL in the
```

Comment:
definitions

Reply:
no, definitinos is the Italian with a precise meaning of "this is all bullshit" :) (I'll fix)